packages/@aws-cdk-testing/framework-integ/test/aws-lambda-nodejs/test/integ.dependencies-bun.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from 'path';
+import { Runtime } from 'aws-cdk-lib/aws-lambda';
+import * as cdk from 'aws-cdk-lib';
+import { ExpectedResult, IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as lambda from 'aws-cdk-lib/aws-lambda-nodejs';
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'TestStack');
+
+const handler = new lambda.NodejsFunction(stack, 'Function', {
+  entry: path.join(__dirname, 'integ-handlers/bun/dependencies-bun.ts'),
+  runtime: Runtime.NODEJS_20_X,
+  bundling: {
+    minify: true,
+    // Will be installed, not bundled
+    // (axios is a package with sub-dependencies,
+    // will be used to ensure bun bundling works as expected)
+    nodeModules: ['axios'],
```

Comment:
Please let me know: If axios is not listed in bun.lockb, will specifying it in `nodeModules` still ensure it is installed and works correctly?

If that is the case, I think it would be better to run an integration test to ensure that a Lambda function using modules listed only in bun.lockb works correctly.

Reply:
This is what's also done in the PNPM integration test: https://github.com/aws/aws-cdk/blob/81cde0e2e1f83f80273d14724d5518cc20dc5a80/packages/%40aws-cdk-testing/framework-integ/test/aws-lambda-nodejs/test/integ.dependencies-pnpm.ts#L16-L19

Axios is listed in `bun.lockb`, but specifying it in `nodeModules` installs it using `bun` inside the docker container ([docs](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda_nodejs.BundlingOptions.html#nodemodules)). I verified that the sub-dependencies (locked by `bun.lockb`) were installed properly.